interoperability-tests/Testing-between-Gramine-and-IntelSGXSDK-ratls.md

```diff
@@ -0,0 +1,158 @@
+Testing between Gramine and rats-tls
+====
+
+# Introduction
+
+This document records the interoperability testing between [Gramine](https://github.com/gramineproject/gramine) and [Intel SGXSDK](https://github.com/intel/linux-sgx), both of which have implemented Interoperable RA-TLS.
+
+We conducted two sets of tests, in which Gramine and Intel SGXSDK ra-tls respectively verified each other's certificates and established TLS secure channels. You can verify our tests by following the steps below.
+
+# Prerequisites
+
+## Intel SGXSDK 
+
+To run the tests, first you need to download and install Intel SGXSDK and corresponding PSW packages on your system. 
+
+Interoperable RA-TLS was implemented since Intel SGXSDK 2.22 release. 
+Download the software package from following link:
+https://download.01.org/intel-sgx/latest/linux-latest/distro/
+
+Pick up your the distro for your OS. 
```

Comment:
I don't like these three lines:
- First of all, they are obvious.
- Second, you bind to a particular version of Intel SGX SDK (`2.22.100.3`).

Maybe just remove these three lines? Instead of these three lines, you can give a link to the Installation Guide of Intel SGX SDK/PSW.